project/plugins.sbt

```diff
@@ -1,7 +1,7 @@
-addSbtPlugin("ch.epfl.scala" % "sbt-release-early" % "1.2.0")
+addSbtPlugin("ch.epfl.scala" % "sbt-release-early" % "2.0.0")
 addSbtPlugin("io.get-coursier" % "sbt-coursier" % "1.0.0-RC13")
-addSbtPlugin("com.eed3si9n" % "sbt-buildinfo" % "0.6.1")
+addSbtPlugin("com.eed3si9n" % "sbt-buildinfo" % "0.7.0")
 addSbtPlugin(
-  "com.thesamet" % "sbt-protoc" % "0.99.12" exclude ("com.trueaccord.scalapb", "protoc-bridge_2.10")
+  "com.thesamet" % "sbt-protoc" % "0.99.13" exclude ("com.trueaccord.scalapb", "protoc-bridge_2.12")
```

Comment:
I'm note sure what this exclusion is for and whether it's still needed.

Reply:
Doesn't seem needed anymore https://scalapb.github.io/sbt-settings.html it used to be recommended in the official docs